src/main/java/org/LogAggregatorTool/operations/ThreadCreation.java

```diff
@@ -0,0 +1,19 @@
+package org.LogAggregatorTool.operations;
+
+import java.io.IOException;
+import java.util.concurrent.Callable;
+
+public class ThreadCreation implements Callable<String> {
+    private String pathToLogFilesFolder;
+    private String outputFilePath;
+
+    public ThreadCreation(String pathToLogFilesFolder,String outputFilePath){
+        this.pathToLogFilesFolder=pathToLogFilesFolder;
+        this.outputFilePath=outputFilePath;
+    }
+    @Override
+    public String call() throws IOException {
+        LogFilesProcessor logFilesProcessor=new LogFilesProcessor();
```

Comment:
This can be directly used in LogFilesExecutor as child thread
Now ThreadCreation is only calling Processor and its tightly coupled. 
Either use ThreadCreation as Interface or directly use LogFilesExecutor as child thread